packages/react-composites/src/composites/CallWithChatComposite/PeopleButton.tsx

```diff
@@ -24,7 +24,7 @@ export const PeopleButton = (props: ControlBarButtonProps): JSX.Element => {
         },
         props.styles ?? {}
       ),
-    [theme]
+    [props.styles, theme.palette.neutralLight]
```

Comment:
nit: I tend to think of `theme` as an atomic unit. I'd not expect Contoso to modify parts of the theme so it's OK to add `theme` as the dependency (especially if the list of theme fields we reference starts growing)